Demos/ToDoList.Api/Program.cs

```diff
@@ -49,7 +49,7 @@
 
 #region User Accounts Projections
 
-var userEventSourcingBuilder = builder.Services.AddPostgresqlEventStore(builder.Configuration.GetConnectionString("Default"), "user-events")
+var userEventSourcingBuilder = builder.Services.AddPostgresqlEventStore(builder.Configuration.GetConnectionString("Default"), "user-events", "user-items")
```

Comment:
It's not convenient to pass additional param when it's not needed and it's hard to guess what that means if you don't have much experience with events and items. Let's make it easier by automatically generating items container name from event store name and make that parameter optional. 

Just append "-items" to event store name by default.